api/src/main/java/io/opentelemetry/trace/Span.java

```diff
@@ -283,6 +283,14 @@
    */
   SpanContext getContext();
 
+  /**
+   * Returns the parent {@code Context} associated with this {@code Span}.
+   *
+   * @return the parent {@code Context} associated with this {@code Span}.
+   * @since 0.8.0
+   */
+  Context getParent();
```

Comment:
This is only used in the OpenTracingShim so far and the usage there is questionable, so we could remove this change if we want to minimize the PR.

Reply:
I simplified the PR in e102deff, but the state at a7fccf470dd250441820c43ae00bca840a1fdd48 is also still interesting.